tasks.py

```diff
@@ -246,6 +246,7 @@ def _build_flags(
         "namespace": namespace,
         "client-side-validation": client_side_validation,
         "black": True,
+        "generate-sample": True,
```

Comment:
i'm not sure we want to generate samples for every single test case, I'm thinking we canjust do it for azure/azure-arm

Reply:
got it